pkg/cluster/deploystorage_resources.go

```diff
@@ -42,7 +42,8 @@ func (m *manager) denyAssignment() *arm.Resource {
 							"Microsoft.Compute/snapshots/write",
 							"Microsoft.Network/networkInterfaces/effectiveRouteTable/action",
 							"Microsoft.Network/networkSecurityGroups/join/action",
-							"Microsoft.Resources/tags/*", // Enable tagging for Resources RP only
+							"Microsoft.Resources/tags/*",                      // Enable tagging for Resources RP only
+							"Microsoft.Resources/deployments/validate/action", // Validates whether the specified template is syntactically correct
```

Comment:
we should still be able to validate this potentially in FF INT.  I believe we may have a deny assignment configured there.  Please check and/or talk to CF on this.

Reply:
Already contacted Michael for this, i'll update this PR as I discover more information

Reply:
@bennerv after talking to Michael, and checking both FFInt and FFProd azure portals ( escort session), i verified that there are no deny assignments on FFInt, only on FFProd :(

Reply:
@karanmagdani I think we could test this in a canary region then before rolling it out to the masses.  Sync with me later this week to go through an ad-hoc release process.